src/main/java/com/powsybl/openloadflow/network/AbstractLfGenerator.java

```diff
@@ -65,6 +65,7 @@ public double getMaxRangeQ() {
                         } else {
                             maxRangeQ = Math.max(maxRangeQ, point.getMaxQ() - point.getMinQ());
                         }
+                        maxRangeQ = maxRangeQ / PerUnit.SB;
```

Comment:
In fact, it seems that reactive capability curves are also already in per unit. What do you think ?